metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -150,9 +150,12 @@ def extract_lineage(
     ) -> List[MetadataChangeProposalWrapper]:
         mcps: List[MetadataChangeProposalWrapper] = []
 
+        assert table.dataset
+        parameters = table.dataset.parameters
```

Comment:
So parameters are already provided to us? Nice!

Reply:
not quite - we're making an extra API call to fetch them